src/Spd.Resource.Repository/BizLicApplication/BizLicApplicationRepository.cs

```diff
@@ -162,43 +166,66 @@ private void LinkOrganization(Guid? accountId, spd_application app)
         }
     }
 
-    private void LinkPrivateInvestigator(SwlContactInfo privateInvestigatorInfo, spd_application app)
+    private spd_businesscontact UpsertPrivateInvestigator(PrivateInvestigatorSwlContactInfo privateInvestigatorInfo, spd_application app)
     {
-        if (privateInvestigatorInfo.LicenceId == null)
-            return;
+        spd_businesscontact? bizContact = null;
+        DeletePrivateInvestigatorLink(app);
```

Comment:
why we need to do this?

Reply:
The idea is to remove all relationships (links) around business contact before adding or updating a new business contact